spec/controllers/concerns/third_party_transaction_logging_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ThirdPartyTransactionLogging do
+  let!(:test_controller) do
```

Comment:
Like we said, I am not sure if it is better to use `let` or `let!` here, but if the test passes it probably does not matter. Just mentioning it.